### PR TITLE
netty: call transportReady in handleProtocolNegotiationCompleted

### PR DESCRIPTION
**Pull request by dapengzhang0**
implements the comments mentioned in #2619

### CODE REVIEW

**Comment by ejona86** [comment]
CC @lukaszx0, @kkaneda

**Comment by kkaneda** [comment]
Thanks! LGTM.

File: netty/src/main/java/io/grpc/netty/NettyServerHandler.java [hunk 1]
```diff
@@ -41,8 +41,8 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
+
```
Comment (ejona86) [inline netty/src/main/java/io/grpc/netty/NettyServerHandler.java]: Remove the newlines; that's not the new style. Ditto elsewhere.

File: netty/src/main/java/io/grpc/netty/ProtocolNegotiator.java [hunk 2]
```diff
@@ -53,7 +53,8 @@
 
   /**
    * Creates a new handler to control the protocol negotiation. Once the negotiation
-   * has completed successfully, the provided handler is installed.
+   * has completed successfully, the provided handler is installed. Must call {@code
+   * grpcHandler.onHandleProtocolNegotiationCompleted()} at certain point.
```
Comment (ejona86) [inline netty/src/main/java/io/grpc/netty/ProtocolNegotiator.java]: If the negotiation fails, it isn't necessary to call the completed callback.

**Pull request merged**